introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-transfer-learning-highlevel.ipynb

```diff
@@ -9,9 +9,16 @@
     "1. [Introduction](#Introduction)\n",
```

Comment:
<p>Is this formatted correctly? Looks odd in ReviewNB but may just be another case of ReviewNB formatting. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
It is 

![Screenshot 2022-11-09 at 19 07 24](https://user-images.githubusercontent.com/2484268/200919255-b2207966-9fac-475a-8325-2000e2d0d57b.png)

Btw, I fixed a few tables of contents that were not properly reflecting the structure of the notebooks.